api/agentlocalpb/agentlocal.proto

```diff
@@ -33,6 +33,7 @@ message AgentInfo {
   // The current listen port of this Agent (exporter or vmagent).
   // Zero for other Agent types, or if unknown or not yet supported.
   uint32 listen_port = 4;
+  string path = 5;
```

Comment:
it probably should not be in AgentInfo, because it's not related to pmm-agent itself.
Just a path doesn't look a good naming
Please add comment.